terraform/a2ha-terraform/modules/automate/files/config.toml.erb

```diff
@@ -95,6 +95,79 @@
   port = <%= overrides[:teams_port] %>
 <% end %>
 
+<% if overrides[:backup_config_s3] == "true" -%>
+[global.v1.external.elasticsearch.backup]
+  enable = true
+  location = "s3"
+
+[global.v1.external.elasticsearch.backup.s3]
+
+  # bucket (required): The name of the bucket
+  bucket = "<%= overrides[:bucket_name] %>"
+
+  # base_path (optional):  The path within the bucket where backups should be stored
+  # If base_path is not set, backups will be stored at the root of the bucket.
+  base_path = "elasticsearch"
+
+  # name of an s3 client configuration you create in your elasticsearch.yml
+  # see https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-client.html
+  # for full documentation on how to configure client settings on your
+  # Elasticsearch nodes
+  client = "default"
+
+[global.v1.external.elasticsearch.backup.s3.settings]
+  ## The meaning of these settings is documented in the S3 Repository Plugin
+  ## documentation. See the following links:
+  ## https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-repository.html
+
+  ## Backup repo settings
+  # compress = false
+  # server_side_encryption = false
+  # buffer_size = "100mb"
+  # canned_acl = "private"
+  # storage_class = "standard"
+  ## Snapshot settings
+  # max_snapshot_bytes_per_sec = "40mb"
+  # max_restore_bytes_per_sec = "40mb"
+  # chunk_size = "null"
+  ## S3 client settings
+  # read_timeout = "50s"
+  # max_retries = 3
+  # use_throttle_retries = true
+  # protocol = "https"
+[global.v1.backups]
+  location = "s3"
+[global.v1.backups.s3.bucket]
+  # name (required): The name of the bucket
+  name = "<%= overrides[:bucket_name] %>"
+
+  # endpoint (required): The endpoint for the region the bucket lives in.
+  # See https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region
+  endpoint = "<%= overrides[:s3_endpoint] %>"
+
+  # base_path (optional):  The path within the bucket where backups should be stored
+  # If base_path is not set, backups will be stored at the root of the bucket.
+  base_path = "automate"
+
+[global.v1.backups.s3.credentials]
```

Comment:
remove this access key secret key config

Reply:
Credential block is removed.